### PR TITLE
Fix padding on program card

### LINKED ISSUE #3241: Remove top padding from StartTimeWrapper and add as bottom padding to ProgramCard in Radio Schedule
**Is your feature request related to a problem? Please describe.**
We have varying bottom margins (16px and 24px) between the section label and radio programs for different breakpoints. So this should be added in simorgh where the SectionLabel and RadioSchedule components are composed.

**Describe the solution you'd like**

- Remove the top padding from the `StartTimeWrapper`.
- Add the padding as padding-bottom to the `ProgramCard`
Zeplin link: https://app.zeplin.io/project/5c8a5e021d454124ef90669f/screen/5e1cb5109da20e0404f7f0ec

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by EinsteinNjoroge**
Resolves #3241

**Overall change:** _Remove top padding from `StartTimeWrapper` and add as bottom padding to `ProgramCard` in Radio Schedule ._

**Code changes:**

- _Remove top padding from StartTimeWrapper and add as bottom padding to ProgramCard in Radio Schedule._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-radio-schedule/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 0.1.0-alpha.20 | [PR#3197](https://github.com/bbc/psammead/pull/3197) Add padding bottom to program cards  |
```
Comment (EinsteinNjoroge) [inline packages/components/psammead-radio-schedule/CHANGELOG.md]:
```suggestion
| 0.1.0-alpha.20 | [PR#3242](https://github.com/bbc/psammead/pull/3242) Add padding bottom to program cards  |
```

**Review by itimirichard — APPROVED** [review]
LGTM, thanks!

**Review by thekp — APPROVED** [review]
image
LGTM 💯

**Comment by EinsteinNjoroge** [comment]
This fix will be tested as part of https://github.com/bbc/simorgh/issues/4671

**Pull request merged**